esphome/components/api/api_server.cpp

```diff
@@ -136,16 +155,24 @@ void APIServer::loop() {
     }
   }
 }
+
 void APIServer::dump_config() {
   ESP_LOGCONFIG(TAG, "API Server:");
-  ESP_LOGCONFIG(TAG, "  Address: %s:%u", network::get_use_address().c_str(), this->port_);
+  ESP_LOGCONFIG(TAG, "  Address: %s: %u", network::get_use_address().c_str(), this->port_);
```

Comment:
I think I added this space but it shouldn't be here:
```suggestion
  ESP_LOGCONFIG(TAG, "  Address: %s:%u", network::get_use_address().c_str(), this->port_);
```